pytest_mh/utils/fs.py

```diff
@@ -527,3 +527,20 @@ def wc(
             args.append("-m")
 
         return self.host.ssh.exec(["wc", *args, file])
+
+    def chmod(self, mode: str, path: str, args: list[str] = []) -> SSHProcessResult:
+        """
+        Change file mode bits.
+        Mode can be specified in two ways: octal number e.g. "666", "444" or
+        a symbolic representation of changes e.g. "u=rw,go=r", "go-rw"
+
+        :param mode: New mode of file
+        :type mode: str
+        :param path: Folder whose permissions change
```

Comment:
This could also be a file, so rephrase the comment so that it's more generic or add `file` to it